src/network/graphql/mutations.js

```diff
@@ -1,8 +1,8 @@
 import gql from 'graphql-tag';
 import _ from 'lodash';
 
-import client from './graphClient';
-import { TYPE, getMutation, isValidEnum } from './graphSchema';
+import client from '.';
```

Comment:
`import client from './index';` was surprised to see that lol

Reply:
oh it used to be called `graphClient`